source/bsondump-compatibility-and-platform-support.txt

```diff
@@ -0,0 +1,53 @@
+.. _bsondump-compatibility-and-platform-support:
+
+===============================================
+``bsondump`` Compatibility and Platform Support
+===============================================
+
```

Comment:
You might consider adding installation on this page. Compatibility and installation are somewhat related.